_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,412 @@
+---
+id: 9bd38c
+name: Non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when content is identified through a visual reference, there are also non-visual references identifying the same content.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - Language
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Daniël Strik
+    - Wilco Fiers
+---
+
+## Applicability
+
+Any [text node](https://dom.spec.whatwg.org/#text) that includes one of the [visual reference words](#visual-reference-words), that is [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree)
+
+## Expectation
+
+Each test target that describes any [web content](https://www.w3.org/TR/WCAG21/#dfn-content) on this website, through the use of the [visual reference words](#visual-reference-words), is on the same [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s) with a [textual](https://www.w3.org/TR/WCAG21/#dfn-text) instruction that also identifies that [web content](https://www.w3.org/TR/WCAG21/#dfn-content) by a non-visual characteristic, except if:
```

Comment:
I am afraid we can't do this. Because you can use visual reference words to describe something by a non-visual characteristic. For example "below" can be used to describe "below this content in the dom-order" which is not a visual characteristic.

Reply:
My ambiguity concern comes from "characteristic". What exactly is considered a characteristic?

Reply:
Anything really. XD

Would "property" be better in your opinion?

Reply:
No, I don't think "property" is better. Let's leave it as it is.